Automattic/wp-calypso
# Videos UI: Accept a full header component as a prop.

To simplify the amount of props and prop drilling needed for the header and footer bars, while increasing versatility, this PR separates out the header bar into its own standalone component for the modal context, and then passes the entire component into `<VideosUi />` as a prop.

**Testing Instructions**
* Use the calypso.live link and go to a site's My Home.
* Open the videos UI by clicking "Start learning" on the "Blog like an expert from day one" educational card.
* Verify the "close" cross in the top right corner works properly, and is not present on mobile widths.
